packages/grafana-ui/src/components/Monaco/types.ts

```diff
@@ -21,6 +24,8 @@ export interface CodeEditorProps {
    */
   onEditorDidMount?: (editor: any) => void;
```

Comment:
In the "real work" I want to type the `editor` argument so I can be confident it don't screw it up :) :)